lib/screens/login/view/login_screen.dart

```diff
@@ -203,6 +205,10 @@ class LoginScreenState extends State<LoginScreen>
     ];
   }
 
+  bool showBiometrics() {
+    return (!globals.testingActive && Platform.isIOS) || Platform.isAndroid;
```

Comment:
We should avoid polluting the code with test logic. Is there any other way to know if the biometrics are enabled? If not, I'd move this to an external function such as `isBiometricsEnabled` which handles that logic.

Reply:
yeah, we check for that in [lib/screens/login/bloc/login_bloc.dart#L39](https://github.com/witnet/my-wit-wallet/blob/main/lib/screens/login/bloc/login_bloc.dart#L39)
i think there is an issue / edge case if the host device supports biometrics and is running the e2e tests via an emulator.

Reply:
We don't need to check if biometrics are enabled, we need to check if there is a testing environment to disable it, otherwise it will show the native popup and the e2e testing will break in iOS.